pint_xarray/tests/test_conversion.py

```diff
@@ -459,3 +459,10 @@ def test_strip_unit_attributes(self, obj, expected):
         assert (
             filter_none_values(conversion.extract_unit_attributes(actual)) == expected
         )
+
+
+class TestParsing:
+    def test_parse_integer_inverse(self):
+        da = DataArray([10], attrs={'units': 'm^-1'})
+        result = da.pint.quantify()
+        assert result.pint.units == pint.Unit('1 / meter')
```

Comment:
the change was in `pint_xarray.accessors`, could you move this test to `xarray.tests.test_accessors`?